ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -505,11 +506,15 @@ class RootViewController: UIViewController, UITableViewDataSource, UITableViewDe
         }
         for item in items {
             if item.tag == BarButtonItemTag.dismiss.rawValue {
-                item.badgeValue = isOn ? "12345" : nil
+                let badgeValue = isOn ? "12345" : nil
+                let badgeAccessibilityLabel = isOn ? "items" : nil
+                item.setBadgeValue(badgeValue, badgeAccessibilityLabel: badgeAccessibilityLabel)
             } else if item.tag == BarButtonItemTag.threeDay.rawValue {
-                item.badgeValue = isOn ? "12" : nil
+                let badgeValue = isOn ? "12" : nil
+                item.setBadgeValue(badgeValue, badgeAccessibilityLabel: nil)
             } else {
-                item.badgeValue = isOn ? "New" : nil
+                let badgeValue = isOn ? "New" : nil
+                item.setBadgeValue(badgeValue, badgeAccessibilityLabel: nil)
```

Comment:
I think the benefit of setting the badgeAccessibilityLabel is to customize in case the value is not a number.
Would it be nice to showcase the accessibility label with a custom one here (like "New feature" or something along those lines)?

Reply:
Done!!